### DISCUSSION TITLE
Version 1.15.0

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#162] [Announcements]
- Kvæsitso can now be set as assistant app
- grid labels can now be hidden
- added an option to make the clock widget not fill the entire screen height
- added teardrop icon shape
- fix app incorrect app widget sizing
- bugfixes and other minor improvements

Furthermore, there is good news if you have previously had issues connecting your Google account: the app now no longer appears as "unverified" and the 100 user cap has been lifted. Google signin should now work again for all users.

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.15.0'>Version 1.15.0</a>.</em>

**Comment by Emma-pete** [comment]
Does the update of this app comes up on the app itself or one has to go through the site to download the update?
